practices/java-web/HelloWorldFilterWebApplication/src/main/java/com/dowa/java/web/filters/StatisticFilter.java

```diff
@@ -0,0 +1,43 @@
+package com.dowa.java.web.filters;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+/**
+ * Created by nexuz on 31/05/15.
+ */
+public class StatisticFilter implements Filter {
+    private int indexVisitNum = 0;
```

Comment:
this is too specific. What happens If I add another page to the web-app? I would need to come here and add another counter for that page, that violates open/close principle. Use a map instead code for generic solutions